src/lib/storage/table.cpp

```diff
@@ -17,57 +17,112 @@
 
 namespace opossum {
 
-Table::Table(const ChunkOffset target_chunk_size) {
-  // Implementation goes here
+Column::Column(std::string name, std::string type) : name(std::move(name)), type(std::move(type)) {}
```

Comment:
`Column` makes me think of a container that actually holds the data and not only the definition. In Hyrise we have a similar type `TableColumnDefinition`.